lib/Configuration.js

```diff
@@ -15,7 +15,7 @@ const DefaultConfig = {
   packagerOptions: {},
   keepOutputDirectory: false,
   config: null,
-  serializedCompile: false
+  concurrency: Infinity
```

Comment:
So the default value is infinity? Shouldn't this be _to the number of available cores_?

Reply:
You're correct.